prepareJWTSessionRSA

```diff
@@ -0,0 +1,20 @@
+func prepareJWTSessionRSA(testName string) (*APISpec, string) {
```

Comment:
these should not be committed to repo

Reply:
Yes, thanks, buger had mentioned that on the main thread